requirements.txt

```diff
@@ -2,3 +2,5 @@ gkeepapi
 keyring
 notion-client>=2.0.0
 requests<2.30.0
+notion
+tqdm
```

Comment:
This one is not used either. I guess you intended to add a progress bar but it didn't make it to the final version 😄